R/license.R

```diff
@@ -72,6 +72,22 @@ use_agpl3_license <- function(name = find_name()) {
   use_license_template("AGPL-3", name)
 }
 
+#' @rdname licenses
+#' @export
+use_agpl3_license <- function(name = find_name()) {
+  force(name)
+  check_is_package("use_agpl3_license()")
+
+  use_description_field("License", "AGPL-3 + file LICENSE", overwrite = TRUE)
+  use_license_template("AGPL-3", name)
+
+  use_template(
+    "year-copyright.txt",
+    save_as = "LICENSE",
+    data = license_data(name)
+  )
+}
+
```

Comment:
Why is this part of the diff? It was added in the previous PR, right?

Reply:
@jennybc the previous PR was merged, now I editerd directly onm github.com and I added an "a" to the url, therefore it fixes the link